tests/components/filter/test_sensor.py

```diff
@@ -27,27 +28,27 @@
     STATE_UNKNOWN,
     UnitOfTemperature,
 )
-import homeassistant.core as ha
+from homeassistant.core import HomeAssistant, State
 from homeassistant.helpers import entity_registry as er
 from homeassistant.setup import async_setup_component
 import homeassistant.util.dt as dt_util
 
 from tests.common import assert_setup_component, get_fixture_path
 
 
-@pytest.fixture
-def values():
+@pytest.fixture(name="values")
+def values_fixture() -> list[State]:
     """Fixture for a list of test States."""
     values = []
     raw_values = [20, 19, 18, 21, 22, 0]
     timestamp = dt_util.utcnow()
     for val in raw_values:
-        values.append(ha.State("sensor.test_monitored", val, last_updated=timestamp))
+        values.append(State("sensor.test_monitored", str(val), last_updated=timestamp))
```

Comment:
The `state` argument is expected to be a string, so I've adjusted it here to better reflect the reality.

https://github.com/home-assistant/core/blob/a44e44b7d01c2a10e0b84e22fbd2b20012024159/homeassistant/core.py#L1129